tests/sources/tools/perception/speech_recognition/matchboxnet/test_matchboxnet.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 1996-2020 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import shutil
+import unittest
+from urllib.request import urlretrieve
+
+import librosa
+import numpy as np
+import torch as t
+
+from opendr.engine.constants import OPENDR_SERVER_URL
+from opendr.perception.speech_recognition.matchboxnet.matchboxnet_learner import MatchboxNetLearner
+from opendr.engine.data import Timeseries
+from opendr.engine.datasets import DatasetIterator
+from opendr.engine.target import Category
+
+TEST_BATCH_SIZE = 2
+TEST_EPOCHS = 1
+TEST_CLASSES_N = 20
+TEST_INFER_LENGTH = 2
+TEST_SIGNAL_LENGTH = 16000
+
+TEMP_SAVE_DIR = os.path.join(".", "tests", "sources", "tools", "perception", "speech_recognition",
+                             "matchboxnet", "matchboxnet_temp")
+
+
+class DummyDataset(DatasetIterator):
+    def __init__(self):
+        super().__init__()
+
+    def __len__(self):
+        return TEST_BATCH_SIZE * 2
+
+    def __getitem__(self, item):
+        return np.ones(TEST_SIGNAL_LENGTH), np.random.choice(TEST_CLASSES_N)
+
+
+class MatchboxNetTest(unittest.TestCase):
+    learner = None
+
+    @classmethod
+    def setUpClass(cls):

```

Comment:
Could you please add a log message specifying which test is running?
Something like this:
```suggestion
    def setUpClass(cls):
        print("\n\n**********************************\nTEST Speech command recognition MatchboxNetLearner\n"
              "**********************************")
```